peripherals/usart/_v2_AB_common.yaml

```diff
@@ -0,0 +1,21 @@
+# Common fields between v2 A and B peripherals
+
+CR1:
+  UESM:
+    Disabled: [0, "USART not able to wake up the MCU from Stop mode"]
+    Enabled: [1, "USART able to wake up the MCU from Stop mode"]
+CR3:
+  WUFIE:
+    Inhibited: [0, "Interrupt is inhibited"]
+    Interrupted: [1, "An USART interrupt is generated whenever WUF=1 in the ISR register"]
+  WUS:
+    Address: [0, "WUF active on address match"]
+    Start: [2, "WuF active on Start bit detection"]
```

Comment:
Should be `WUF`

Reply:
I have forget about it, will do.

Reply:
Sorry, I wasn't clear, I was referring to the lower-case `u` in `WuF`. Though checking the RM more carefully I see it makes the same typo. Still I think we should probably write `WUF` since `WuF` doesn't make sense.